examples/IBFE/explicit/ex0/input2d.test

```diff
@@ -122,6 +123,7 @@ IBFEMethod {
    use_jump_conditions        = USE_JUMP_CONDITIONS
    use_consistent_mass_matrix = USE_CONSISTENT_MASS_MATRIX
    IB_point_density           = IB_POINT_DENSITY
+   Phi_dt                     = DT
```

Comment:
The code need to find out the time step size internally and not from the input file --- want to make it easy to use dynamic time step sizes whenever possible.

Reply:
Just wanted to re-up this comment.

Reply:
It seems like some of the protected functions in ```HierarchyIntegrator``` give me access to some timestep info.  What is the best way to access these in ```IBFEMethod```?